payments/swish/other-features.md

```diff
@@ -18,20 +18,32 @@ sidebar:
 
 {% include alert-development-section.md %}
 
-{% include settlement-reconciliation.md %}
-
-{% include one-click-payments.md %}
+## Payment Resource
 
-{% include payment-link.md %}
+{% include payment-resource.md payment-instrument="swish" %}
 
-{% include recurring-card-payments.md %}
+{% include settlement-reconciliation.md %}
 
-{% include subsite.md payment-instrument="swish" %}
+{% include payment-link.md %}
 
 ### Prices
 
 {% include prices.md %}
```

Comment:
This include has URI's with `creditcard` value. Maybe add variable tags that can allows us to change the values based on instrument?